### PR TITLE
Replace x11glvnd with GLX_EXT_libglvnd extension

### PR DESCRIPTION
**Pull request by kbrenneman**
This replaces the x11glvnd X extension with a new GLX extension, based on the suggestions in pull request #76.

To look up the vendor name for a screen, libGLX now sends a glXQueryServerString request with a new GLX_VENDOR_NAMES_EXT enum.

To look up the screen number for a drawable, it'll send a glXGetDrawableAttributes request, and the server will add a GLX_SCREEN attribute to the reply.

### CODE REVIEW

File: doc/GLX_EXT_libglvnd.txt [hunk 1]
```diff
@@ -0,0 +1,186 @@
+Name
```
Comment (aritger) [inline doc/GLX_EXT_libglvnd.txt]: I'd be inclined to include the extension spec in glvnd; it seems likely to grow stale relative to the copy that will eventually be pushed to Khronos.

**Comment by kbrenneman** [comment]
You mean you'd be inclined to include it or to leave it out?

**Comment by aritger** [comment]
Sorry, I meant "I'd be inclined to *not* include the extension spec in glvnd..."

**Comment by kbrenneman** [comment]
Okay. It's easy enough to take out that commit before checking anything in.

**Comment by aritger** [comment]
Thanks.  The rest of the series looks reasonable to me.

**Comment by nwnk** [comment]
Series looks good to me too.

**Comment by kbrenneman** [comment]
I've got an updated version with GLX_EXT_libglvnd.txt removed as pull request #79. Closing this one.

**Pull request closed without merging**